config/credentials.yml.enc

```diff
@@ -1 +1 @@
-u4vSs9rRKgFB9RgWv9p02XEvtP7XQ8zGXKUA4t7UyHCDIdHcoHmpSdjGSTUMScwW8jkTH6lUCVM9ZivZO0ds4o137PIRrZlIOTAVBRNRndgI3ZssOOhG5bkQhEzV5Aa3PhpNxrR4skUiE1VaTDDgZrCipez/63NAuO00G3p/KO2UrxyCyGT5IsNnyb3fPDAD9OiaVF/g1us4rrARgvwaUGB++8+QgPxFSMSdwt18WojUSj+j8b5+XgX36XxngdgnCF9Na3oSnH9HYvM+FzrjyubTUCj2t66Ll3vme4yIEufxWEkcFiv/2f4raFhd4fyEkR3f6tFP7F86UmBAbW5w3Ub+Mvt8OgLpy+TkvchOQGHOs7Myo2MFfE6zscFbOJU5GzgJl8v6faXiz1fn11DVOnNeidKN8RJEAUcD--gw8snR+zohnRIwCV--Orm93ffGg+Aq83DtSpdCAQ==
\ No newline at end of file
+j7MeyJH1xea0QCkb4TtzgYXytdhJHx17Nc5uUGTohXtaJ1e61K948piG4V2ZWQYapo2NK/3SBmXUP/DshSPdDPst8mcGthUwXYqBAPKOXapNanfsjAiu9s2rbKkqGAYiOuT8LWMk+LSqGdI0HAp49viXsicWjeBvTlCOwpiuCipZyfHOOck9N0/BFsU385uhigJ7Zq+F81HwtIWaAIM5b/TEi6uvPBwXw8AmesagLguWGJ/L5HIG5TSw9evo/r76zLlgzskFzCWATePKjy6w/ok7jFAyttrwUgUHgcKz4BqMopTYycul98vrtPp1vcVVUGUqrwCvA4dD+PYTbWXnNjZov4sHWVntUXmoNhbfBSdEfOYQtKeVB1ysWwxVfy7AfoGrx5QLVlD7bGEA9EAfB3BnfljEOLx45zYI4wVJjA4o20QZo7xoV+LidNs5xdkOvowLnEIeaz6RWjGfnGrzRsOFFVHH7mYz--Q46QUOmQziEbkcnf--2OgwWbr3RA6N/AkOO4EKnw==
```

Comment:
EOF

Reply:
not work if new line
![image](https://user-images.githubusercontent.com/35836663/63963002-aeed3180-ca9c-11e9-8e8c-00e43139d17a.png)